src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas2/SubmissionsController.kt

```diff
@@ -30,20 +30,19 @@ import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas2.Application
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas2.OffenderService
```

Comment:
There's a "[LINK TO PR]" still in the commit message here if you still feel there's value in including it.

Reply:
good spot, link is in message now